integreat_cms/cms/views/analytics/translation_coverage_view.py

```diff
@@ -44,67 +44,41 @@ def get(self, request, *args, **kwargs):
 
         region = request.region
 
-        total_number_pages = region.pages.count()
-
-        translation_coverage_data = []
-        outdated_word_count = {}
+        translation_coverage_data = {}
+        outdated_word_count = Counter()
 
+        pages = region.pages.all().prefetch_translations()
```

Comment:
```suggestion
        pages = (
            region.pages.filter(explicitly_archived=False)
            .prefetch_translations()
            .cache_tree()
        )
```
I think it would make sense to only show non-archived pages here.
Unfortunately, we don't have an efficient way to get all non-archived pages at once apart from `cache_tree()` (`region.get_pages(archived=False` doesn't perform well) at the moment, so I'd go with this approach even if it does a little bit of unnecessary tree structure caching...

Reply:
If I understand correctly, these pages should be filtered later using e.g. `if not page.archived:`?